RESUME/api_views.py

```
@@ -17,365 +20,316 @@
+        payload = request.data
+        payload['user'] = self.authenticated_user.id
+        payload['resume_section'] = new_resume_section.id
+        experience_ser: ExperienceUpsertSerializer = ExperienceUpsertSerializer(data=request.data)
+        if experience_ser.is_valid():
+            new_experience = experience_ser.save()
+            return Response(ExperienceFullSerializer(new_experience).data, status=status.HTTP_201_CREATED)
+        else:
+            if new_resume_section:
+                new_resume_section.delete()
+            return ErrorResponse(
+                code=ErrorCode.INVALID_REQUEST, message=f'Invalid Data provided!',
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)